src/Faker/Calculator/TCNo.php

```diff
@@ -12,41 +14,28 @@ class TCNo
      *
      * https://en.wikipedia.org/wiki/Turkish_Identification_Number
      *
+     * @deprecated Use {@link \Faker\Provider\tr_TR\Person::tcNoChecksum()} instead
+     * @see \Faker\Provider\tr_TR\Person::tcNoChecksum()
+     *
      * @param string $identityPrefix
      * @return string Checksum (two digit)
      */
     public static function checksum($identityPrefix)
     {
-        if (strlen((string) $identityPrefix) !== 9) {
-            throw new InvalidArgumentException('Argument should be an integer and should be 9 digits.');
-        }
-
-        $oddSum = 0;
-        $evenSum = 0;
-
-        $identityArray = array_map('intval', str_split($identityPrefix)); // Creates array from int
-        foreach ($identityArray as $index => $digit) {
-            if ($index % 2 == 0) {
-                $evenSum += $digit;
-            } else {
-                $oddSum += $digit;
-            }
-        }
-
-        $tenthDigit = (7 * $evenSum - $oddSum) % 10;
-        $eleventhDigit = ($evenSum + $oddSum + $tenthDigit) % 10;
-
-        return $tenthDigit . $eleventhDigit;
+        return \Faker\Provider\tr_TR\Person::tcNoChecksum($identityPrefix);
     }
 
     /**
      * Checks whether a TCNo has a valid checksum
      *
+     * @deprecated Use {@link \Faker\Provider\tr_TR\Person::tcNoIsValid()} instead
```

Comment:
Please be consistent in where you place the `@deprecated` (after `@return`)

Reply:
Looks better now, thanks.